src/scale_audio_rating.py

```diff
@@ -316,6 +315,7 @@ async def main(cfg, args):
                 "attachments": attachments,
                 "metadata": metadata
             }
+            task_obj['metadata']["file_urls"] = {key: file}

```

Comment:
This causes a bug: Since task_obj["metadata"] is set to be a _reference_ to the metadata, any change in file_urls will affect every task_obj. Previously this was not an issue, since metadata was not being modified, but now it is, since file_urls is changed. Importing copy and replacing "metadata": metadata with "metadata": copy.deepcopy(metadata) fixes the issue.